packages/sentry-client-web/src/components/List.tsx

```diff
@@ -80,20 +91,29 @@ export class List extends React.Component<IListProps, IListState> {
             return null;
         }
 
-        return (
-            <div style={this.getStyle()}>
-                {services.map((service: any, index: number) => {
-                    return (
-                        <div>
-                            <Service
-                                key={index}
-                                service={service} />
-                        </div>
-                    );
-                })}
-            </div>
-        );
-
+        if (services.length > 0) {
```

Comment:
The code here for rendering servers and services is nearly identical. You could have a component instead that has an interface of an array of `displayEntities`, servers or services. Generalizing the null handling code for servers is probably fine, if not better